internal/plugin/connectors/http/generic/config.go

```diff
@@ -71,17 +64,10 @@ func (c *config) renderedHeaders(
 	return headers, nil
 }
 
-// newConfig takes the raw cfgBytes, unmarshals them into a temporary configYAML
-// struct, and then validates and converts that into a config -- which is what
-// our application wants to work with.
-func newConfig(cfgBytes []byte) (*config, error) {
+// newConfig takes a ConfigYAML, validates it, and converts it into a config --
```

Comment:
`converts it into a config` - should indicate what type of config

Reply:
Ha!  Scroll up 10 lines above this comment.  See line 14.